programs/token/src/instructions/initialize_mint_2.rs

```diff
@@ -36,11 +36,11 @@ impl<'a> InitilizeMint2<'a> {
         let account_metas: [AccountMeta; 1] = [AccountMeta::writable(self.mint.key())];
 
         // Instruction data layout:
-        // -  [0]: instruction discriminator
-        // -  [1]: decimals
-        // -  [2..34]: mint_authority
-        // -  [34..35]: freeze_authority presence flag
-        // -  [35..67]: freeze_authority
+        // -  [0]: instruction discriminator (1 byte, u8)
+        // -  [1]: decimals (1 byte, u8)
+        // -  [2..34]: mint_authority (32 bytes, Pubkey)
+        // -  [34]: freeze_authority presence flag (1 byte, u8)
+        // -  [35..67]: freeze_authority (32 bytes, Pubkey)
```

Comment:
Can we add an `optional` flag there, so the text becomes `(optional, 32 bytes, Pubkey)`?